bundles/AdminBundle/Resources/views/SearchAdmin/Search/Quicksearch/asset.html.twig

```diff
@@ -0,0 +1,22 @@
+{# @var $element \Pimcore\Model\Asset\Image|\Pimcore\Model\Asset\Document|\Pimcore\Model\Asset\Video #}
+{% set previewImage = null %}
+{% set params = { 'id': element.id, 'treepreview': true, 'hdpi': true } %}
+
+{% if element is instanceof('\\Pimcore\\Model\\Asset\\Image') %}
+    {% set previewImage = path('pimcore_admin_asset_getimagethumbnail', params) %}
+{% elseif element is instanceof('\\Pimcore\\Model\\Asset\\Video') and callStatic('\\Pimcore\\Video', 'isAvailable') %}
```

Comment:
`callStatic` is not good DX, better create twig extensions:

`pimcore_video_is_available`
`pimcore_document_is_available`